omod-1.10/src/main/java/org/openmrs/module/attachments/rest/AttachmentResource1_10.java

```diff
@@ -51,6 +52,8 @@
 	AttachmentsContext attachmentsContext = Context.getRegisteredComponent(AttachmentsConstants.COMPONENT_ATT_CONTEXT,
 	    AttachmentsContext.class);
 	
+	ComplexDataHelper complexDataHelper;
```

Comment:
We then initialise this variable as an instance of `ComplexDataHelper1_10`

Reply:
thanks,,i have fixed it